<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CourseTimeDialog</class>
 <widget class="QDialog" name="CourseTimeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>331</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="MainLayout" stretch="1,12">
    <property name="spacing">
     <number>12</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="WeekLayout">
      <item>
       <widget class="QLabel" name="BlankLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="MonLabel">
        <property name="text">
         <string>星期一</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="TueLabel">
        <property name="text">
         <string>星期二</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="WedLabel">
        <property name="text">
         <string>星期三</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="ThuLabel">
        <property name="text">
         <string>星期四</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="FriLabel">
        <property name="text">
         <string>星期五</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="SatLabel">
        <property name="text">
         <string>星期六</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="SunLabel">
        <property name="text">
         <string>星期日</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="TableLayout" stretch="1,7">
      <item>
       <layout class="QVBoxLayout" name="TimeLayout">
        <item>
         <widget class="QLabel" name="T1Label">
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="T2Label">
          <property name="text">
           <string>2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="T3Label">
          <property name="text">
           <string>3</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="T4Label">
          <property name="text">
           <string>4</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="T5Label">
          <property name="text">
           <string>5</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="T6Label">
          <property name="text">
           <string>6</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="T7Label">
          <property name="text">
           <string>7</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="T8Label">
          <property name="text">
           <string>8</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="T9Label">
          <property name="text">
           <string>9</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="T10Label">
          <property name="text">
           <string>10</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="T11Label">
          <property name="text">
           <string>11</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="T12Label">
          <property name="text">
           <string>12</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="CourseTableLayout">
        <item>
         <layout class="QVBoxLayout" name="C1Layout"/>
        </item>
        <item>
         <layout class="QVBoxLayout" name="C2Layout"/>
        </item>
        <item>
         <layout class="QVBoxLayout" name="C3Layout"/>
        </item>
        <item>
         <layout class="QVBoxLayout" name="C4Layout"/>
        </item>
        <item>
         <layout class="QVBoxLayout" name="C5Layout"/>
        </item>
        <item>
         <layout class="QVBoxLayout" name="C6Layout"/>
        </item>
        <item>
         <layout class="QVBoxLayout" name="C7Layout"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="MessageLabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>330</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>请选择上课时间</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CreateButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>340</y>
     <width>80</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>完成</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
